lib/interpreters/split_clustering.rb

```diff
@@ -294,13 +294,15 @@ def self.split_solve_core(service_vrp, job = nil, &block)
         ss_data[:cannot_split_further] = false
         log "<-- split_solve_core level: #{split_level}"
 
-        solutions.reduce(&:+)
+        solution = solutions.reduce(&:+)
+        Cleanse.cleanse_duplicate_empties_fills(service_vrp[:vrp], solution)
```

Comment:
Is there a specific reason for calling this here instead of adding it to the `self.cleanse` function so that it is called systematically?

Reply:
I don't understand, why this forces us to call it here. Can't we keep the duplicate empty/fill till the end of solve (where we call the cleanse method -- if I am not mistaken). If we cannot then, maybe the location of the cleanse method call is not correct either..

Reply:
Cleanse was in the `solve` method, this implies that it was only applied right after the call of the solver.

I've moved the cleanse method to `define_process`. This way, every solution whatever the resolution process (split/dicho/solver) is correct.